packages/expo-modules-autolinking/src/__tests__/ReactImportsPatcher-test.ts

```diff
@@ -43,6 +43,54 @@ describe(patchFileAsync, () => {
 `);
   });
 
+  it('should transform double-quoted import which is indented', async () => {
+    const headerSet = new Set(['RCTBridge.h']);
+    mockFsReadFile.mockResolvedValue(`\
+  #if __has_include("RCTBridge.h")
+    #import "RCTBridge.h"
+  #elif __has_include("React/RCTBridge.h")
+    #import "React/RCTBridge.h"
+  #else
+    #import <React/RCTBridge.h>
+  #endif
+`);
+
+    await patchFileAsync(headerSet, 'someFile.h', /* dryRun */ false);
+    expect(mockFsWriteFile.mock.calls[0][1]).toBe(`\
+  #if __has_include(<React/RCTBridge.h>)
+    #import <React/RCTBridge.h>
+  #elif __has_include(<React/RCTBridge.h>)
+    #import <React/RCTBridge.h>
+  #else
+    #import <React/RCTBridge.h>
+  #endif
+`);
+  });
+
+  it('should transform double-quoted import which has trailing spaces', async () => {
+    const headerSet = new Set(['RCTBridge.h']);
+    mockFsReadFile.mockResolvedValue(`\
+#if __has_include("RCTBridge.h") 
+  #import "RCTBridge.h"  
```

Comment:
There's trailing spaces on these lines but I wasn't sure the best way to highlight that in the test.

It will be highlighted in the jest diff if there's a mismatch but I can see someone accidentally stripping it via some kind of file formatting in future?

Maybe something uglier like `#if __has_include("RCTBridge.h")${'  '}` would be clearer?